doc/source/examples/cd_mmd_keops.ipynb

```diff
@@ -0,0 +1,513 @@
+{
```

Comment:
<p>Worth being more explicit about the GPU requirement to run this notebook? Could add a warning/callout box so it's clear that it's long-running and requires/recommended GPU.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi-detect/pull/548/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
From the introduction:

```text
In this notebook we will run a few simple benchmarks to illustrate the speed and memory improvements from using KeOps over vanilla PyTorch on the GPU (1x RTX 2080 Ti).
```
Not clear enough?